common/src/main/java/com/thoughtworks/go/server/domain/AgentInstances.java

```diff
@@ -74,124 +77,114 @@ public AgentInstance findAgent(String uuid) {
     }
 
     public AgentInstance loadAgentInstance(String uuid) {
-        AgentInstance agentInstance = agentInstances.get(uuid);
+        if (isBlank(uuid)) {
```

Comment:
> I would prefer we do not change the behaviour of this method and keep it as is. Also there are no tests for these changes.

The change we made wherein we check for a blank UUID is made to fix possible NullPointerException in case UUID passed in is NULL. Other than that there is no change in behavior.